Ingress/controllers/gce/main.go

```diff
@@ -56,7 +60,7 @@ var (
 		printed to stdout and no changes are made to your cluster. This flag is for
 		testing.`)
 
-	clusterName = flags.String("gce-cluster-name", "default-cluster-name",
+	clusterName = flags.String("gce-cluster-name", "",
```

Comment:
Yeah, you should probably rename it. I thought it was the actual cluster name passed through via the apiserver.